core/src/validator.rs

```diff
@@ -885,7 +885,7 @@ impl Validator {
                     Some((
                         &identity_keypair,
                         node.info
-                            .tpu(Protocol::UDP)
+                            .tpu(Protocol::QUIC)
```

Comment:
`use_quic` is true, so shouldn't we use the `QUIC` protocol address here for the connection cache?

Reply:
Can we just rip out `use_quic` from master entirely?